core/testcasecontroller/simulation/simulation.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Simulation"""
```

Comment:
Do not introduce meaningless comments, please supplement the comments on the Simulation class in detail.

Reply:
When I delete this section, there will be a warning from pylint, so I add it again.

Reply:
Done. The description of the simulation class has been supplemented.